CHANGELOG.md

```diff
@@ -13,10 +13,13 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 
 
 ## [Unreleased]
+### Removed 
+- account_created, created, owned attributes to DictKey resource
+- account_number, branch_code attributes to PaymentPreview resource
 
-## [2.21.0] - 2023-08-06
```

Comment:
ops, da uma olhada como era o change log antigo pq acho q essa parte já tinha:

## [2.21.0] - 2023-08-06
### Changed
- Transfer account_type parameter to required